backend/app/routers/product.py

```
@@ -82,10 +94,13 @@
     return product
 
 
-@router.get("/products/{product_id}", response_model=schemas.ProductOut)
+@router.get("/products/{product_id}")
 def get_product(product_id: int, db: Session = Depends(get_db)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_product`

(ANN201)